src/model/formulations/CarArr2006/pwlcosts.jl

```diff
@@ -2,6 +2,28 @@
 # Copyright (C) 2020, UChicago Argonne, LLC. All rights reserved.
 # Released under the modified BSD license. See COPYING.md for more details.
 
+"""
+    _add_production_piecewise_linear_eqs!
+
+Ensure respect of production limits along each segment.
+Based on Garver (1962) and Carrión and Arryo (2006),
+which replaces (42) in Kneuven et al. (2020) with a weaker version missing the on/off variable.
+Equations (45), (43), (44) in Kneuven et al. (2020).
+NB: when reading instance, UnitCommitment.jl already calculates difference between max power for segments k and k-1
+so the value of cost_segments[k].mw[t] is the max production *for that segment*.
+
+
+===
+Variables
+* :segprod
+* :is_on
+* :prod_above
```

Comment:
Same comment as above, regarding documenting and enforcing decision variables.

Reply:
Note that we do not need `switch_on` and `switch_off`, we only need `is_on` from `Gar1962.StatusVars`.